Improve ch03-05-control-flow collection looping wording (#3758)
rust-lang/book: src/ch03-05-control-flow.md

#### Looping Through a Collection with `for`

> You could choose to use the `while` construct to loop over the elements of a

Reviewer: Even clearer:

```suggestion
You can also use the `while` construct to loop over the elements of a
```
